test/sascalculator/utest_sas_gen.py

```diff
@@ -241,6 +241,8 @@ def test_debye_impl(self):
         import glob
         pdb_files = glob.glob(os.path.join(os.path.dirname(__file__), 'data/debye_test_files', '*.pdb'))
 
+        np.random.seed(1984)
```

Comment:
```suggestion
        rng = np.random.default_rng(932810983)
```

Reply:
Then on line 251, 269:

```
w = rng.random(size=(coords.shape[1],))
```

Reply:
`Generators` are recommended for new code.

Reply:
They are, but its not new code. Change it if you like.

Reply:
`w` is being passed to an external C function as a memory pointer. I don't think this will work as a generator?

Reply:
@klytje the generator just gets called the same-ish as np.random and makes an array.